program/rust/build.rs

```diff
@@ -1,3 +1,18 @@
+mod build_utils;
+use bindgen::Builder;
+use std::vec::Vec;
+
+
 fn main() {
     println!("cargo:rustc-link-search=../c/target");
+    
+    let borsh_derives: Vec<String> = vec!["BorshSerialize".to_string(), "BorshDeserialize".to_string()];
+
+    //make a parser and to it type, traits pairs
```

Comment:
i think people usually put a space after the // and before the text. Actually I'm kind of surprised that rustfmt doesn't fix this for you.

Reply:
yes, sorry noob Rust developer here. Forgot to run cargo fmt. Once this and the other PR are approved, I will do it in a seperate PR that is just fmt. (and then run it after every commit).

If I run it now the diffs will probably be confusing for reviewers.